### PR TITLE
Regen Ledger v5.0 upgrade proposal

### PR DESCRIPTION
**Pull request by ryanchristo**
This pull request adds the draft proposal and long-form proposal description for the upgrade to Regen Ledger `v5.0`.

Block height estimate: https://www.mintscan.io/regen/blocks/8942600

### CODE REVIEW

**Comment by robert-zaremba** [comment]
If you migrated gov proposals, then I suggest to not make a release with Cosmos SDK 0.46.7. Wait for https://github.com/cosmos/cosmos-sdk/pull/14400 being release (0.46.8).
Otherwise you risk that the new proposals won't be handled correctly by the explorers. Kepler and Mintscan confirmed to work on direct gov/v1 support, but as far as I can see they didn't finish it yet.
More context: https://github.com/cosmos/cosmos-sdk/issues/14334

If you don't want to wait, then you can cherry pick it into a Cosmos SDK fork (I did it at Umee for the latest release). [example](https://github.com/umee-network/umee/blob/main/go.mod#L333)

cc: @clevinson, @aaronc

**Comment by ryanchristo** [comment]
Thanks for flagging @robert-zaremba. I've been keeping an eye on the issue. We're using a fork with amino support for gov and group added to the v0.46.7 release so maybe we should consider pulling this into our fork rather than waiting for v0.46.8.

**Comment by aaronc** [comment]
Yes, pulling into our fork sounds like a good option

**Comment by robert-zaremba** [comment]
@ryanchristo then easiest is to merge (not cherry pick) `e7dd5e0bdc8aabadb0d4ae13bdb143f92c1cdf4c` of cosmos-sdk release/v0.46.x into Regen fork.

File: proposals/2023-01-regen-ledger-v5.0-upgrade/proposal.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "title": "Regen Ledger v5.0 Upgrade",
+  "description": "This is a software upgrade proposal for the upgrade to Regen Ledger v5.0. If passed, this proposal would commit Regen Mainnet to halting the application binary for Regen Ledger v4.1 at approximately 16:00 UTC on January 17th at block height 8942600 and requiring nodes to restart with the application binary using Regen Ledger v5.0. \n\nMore details can be found in the long form proposal: https://github.com/regen-network/governance/tree/main/proposals/2023-01-regen-ledger-v5.0-upgrade",
+  "upgrade-height": 8942600,
+  "upgrade-info": {
+    "binaries": {
+      "linux/amd64": "https://github.com/regen-network/regen-ledger/releases/download/v5.0.0/regen-ledger_5.0.0_linux_amd64.zip?checksum=sha256:edbf5beaa769f971cf6b6f3c5e45cc3f38ac7d5b9dd005cddc821ccc37771155",
+      "linux/arm64": "https://github.com/regen-network/regen-ledger/releases/download/v5.0.0/regen-ledger_5.0.0_linux_arm64.zip?checksum=sha256:30b5cc181a970d957577ccee712371c9e88f5cde511d2abdf4a185aa2c97ab1d",
+      "darwin/amd64": "https://github.com/regen-network/regen-ledger/releases/download/v5.0.0/regen-ledger_5.0.0_darwin_amd64.zip?checksum=sha256:XXX",
```
Comment (ryanchristo) [inline proposals/2023-01-regen-ledger-v5.0-upgrade/proposal.json]: cc @clevinson we need to push a darwin package build and update the checksums txt file.

File: proposals/2023-01-regen-ledger-v5.0-upgrade/README.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
+# Regen Ledger v5.0 Upgrade
+
+This is a software upgrade proposal for the upgrade to Regen Ledger `v5.0`. If passed, this proposal would commit Regen Mainnet to halting the application binary for Regen Ledger `v4.1` at approximately 16:00 UTC on January 17th at block height `8942600` and requiring nodes to restart with the application binary using Regen Ledger `v5.0`.
+
+Validators and node operators can refer to [Upgrade Guide v5.0](https://docs.regen.network/validators/upgrade/v5.0-upgrade.html) for guidelines on performing the upgrade.
```
Comment (clevinson) [inline proposals/2023-01-regen-ledger-v5.0-upgrade/README.md]:
```suggestion
Validators and node operators can refer to [Upgrade Guide v5.0](https://docs.regen.network/validators/upgrades/v5.0-upgrade.html) for guidelines on performing the upgrade.
```

Was getting a 404, this link should work:
https://docs.regen.network/validators/upgrades/v5.0-upgrade.html

File: proposals/2023-01-regen-ledger-v5.0-upgrade/proposal.json (hunk 1 above)
Comment (clevinson) [inline proposals/2023-01-regen-ledger-v5.0-upgrade/proposal.json]:
```suggestion
      "darwin/amd64": "https://github.com/regen-network/regen-ledger/releases/download/v5.0.0/regen-ledger_5.0.0_darwin_amd64.zip?checksum=sha256:26d07f258d489650f0dba059f6d3979f7550bf59514c0ee8f8912cca71bff1c6",
```

Comment (clevinson) [reply proposals/2023-01-regen-ledger-v5.0-upgrade/proposal.json]: @ryanchristo can you download the zip and verify checksum?

Assets uploaded here: https://github.com/regen-network/regen-ledger/releases/tag/v5.0.0

Comment (ryanchristo) [reply proposals/2023-01-regen-ledger-v5.0-upgrade/proposal.json]: tACK and verified matching checksum in txt file

**Review by clevinson — APPROVED** [review]

**Pull request merged**